### ISSUE TITLE
Creation of new release definition fails when a release trigger is defined.

### ISSUE THREAD

**Issue by ntrogh** [Microsoft/vsts-python-api#91]
When creating a new release definition which contains a release trigger (no matter which type), the process fails with the following error: 

***vsts.exceptions.VstsServiceError: Invalid data is passed for artifact source trigger object.***

Trigger definition:

```json
"triggers": [
        {
            "artifactAlias": "_CI",
            "triggerConditions": [],
            "triggerType": 1
        }
    ]
```
It seems only the trigger type is being passed to the VSTS REST API and other properties are being stripped out.

When removing the trigger definition from the new release definition, the creation works successfully.

**Comment by tedchamb** [comment]
Fixed in #92

**Comment by tedchamb** [comment]
The fix is available in 0.1.10
